src/Engine/Audio/SteamAudio.cpp

```diff
@@ -5,24 +5,22 @@ SteamAudio::SteamAudio() {
     environmentalRenderer = nullptr;
 }
 
-SteamAudio::SteamAudio(IPLContext * context, IPLhandle * environment) {
-
+SteamAudio::SteamAudio(IPLContext* context, IPLhandle* environment) {
 }
 
 SteamAudio::~SteamAudio() {
-    if(context != nullptr)
+    if (context != nullptr)
         delete context;
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') keyword 'if' not followed by a single space [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=vera%2B%2B%3AT008)